File: samples/java/Bingo/Server/Configuration/src/main/java/systems/zlink/samples/bingo/server/configuration/SampleTimings.java
package systems.zlink.samples.bingo.server.configuration;

import java.time.Duration;

public final class SampleTimings {
    public static final Duration RequestTimeout = Duration.ofSeconds(30);
    public static final Duration DrawPeriod = Duration.ofMillis(20);

    private SampleTimings() {}
}

File: samples/kotlin/ShoppingMall/Client/src/main/kotlin/systems/zlink/samples/kotlin/shoppingmall/client/configuration/SampleTimings.kt
package systems.zlink.samples.kotlin.shoppingmall.client.configuration

import java.time.Duration

object SampleTimings {
    val RequestTimeout: Duration = Duration.ofSeconds(10)
    val WorkflowTimeout: Duration = Duration.ofSeconds(12)
    val PollDelay: Duration = Duration.ofMillis(100)
}
